package.json

```diff
@@ -1182,18 +1182,21 @@
         "webpack-cli": "^4.6.0"
     },
     "dependencies": {
-        "@azure/arm-appinsights": "^4.0.0",
```

Comment:
Sadly the App Insights SDK hasn't GA'd their Track 2 version. So we need to use an alpha build.